### DISCUSSION TITLE
antd@4 menu会出现组件卸载依旧在setState的情况

### DISCUSSION THREAD

**Discussion by SceneryCN** [ant-design/ant-design#42269] [Q&A] [answered]
项目需要两套Layout我通过react-router的Outlet然后通过没有path会直接通过来适配layout但是当我从一个layout跳转到另一个layout的时候menu组件就会报warning，说是我组件卸载了还在操作，如果我ban掉menu就不会出现问题，即使menu什么配置也没有如果我跳转的时候还是会出现这个问题，虽然不影响使用但是我搞不明白

**Comment by zombieJ** [comment, marked as answer]
https://github.com/ant-design/ant-design/issues/41572#issuecomment-1491449544

**Reply by SceneryCN** [reply]
感谢大哥，我直接不管他了，我看了浏览器的内存也没有问题，他莫名其妙报WARNING唉
